app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -476,6 +479,23 @@ class BrowserTabViewModelTest {
         assertEquals(false, loadingViewState().isLoading)
     }
 
+    @Test
+    fun whenProgressChangesToFinishedAndImprovedTabUxVariantActiveThenUpdateWebViewPreview() {
```

Comment:
Optional: Can we rename this to `whenProgressChangesToFinishedAndImprovedTabUxVariantActiveThenWebViewPreviewGenerated` for readability ?